libs/ui-lib/lib/ocm/api/axiosClient.ts

```diff
@@ -15,22 +15,25 @@ const axiosCaseConverterOptions = {
   },
 };
 
-const getDefaultClient = (withoutConverter = false) => {
+const token = '';
+
+const getDefaultClient = () => {
   const client = axios.create();
   client.interceptors.request.use((cfg) => ({
     ...cfg,
     url: `${process.env.AIUI_APP_API_ROOT || ''}${cfg.url || ''}`,
+    headers: {
+      ...cfg.headers,
+      Authorization: `Bearer ${token}`,
+    },
   }));
-  if (withoutConverter) {
-    return client;
-  } else {
-    return applyCaseMiddleware(client, axiosCaseConverterOptions);
-  }
+  return client;
 };
 
-let client: AxiosInstance = getDefaultClient();
-let clientWithoutConverter: AxiosInstance = getDefaultClient(true);
+let client = applyCaseMiddleware(getDefaultClient(), axiosCaseConverterOptions);
+let clientWithoutConverter = getDefaultClient();
```

Comment:
@ammont82 please remind me why do we need an Axios instance without the case converter?

Reply:
Yes, it'ss used in OCM.

Reply:
I'll revert, I did a small refactor while I was looking for the way to make the Cypress tests trigger the `isInOcm` flag.